pkg/qdr/amqp_mgmt.go

```diff
@@ -5,11 +5,10 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
+	amqp "github.com/interconnectedcloud/go-amqp"
```

Comment:
Just out of curiousity, what is the thinking on changing the import order here?

Reply:
I can revert that, this is my IDE "organizing" imports when I opened it for some reason.

Reply:
No need to revert, I was just asking whether it was an explicit choice.